client/src/domain/event/EventInfo.tsx

```diff
@@ -157,7 +161,12 @@ const EventInfo: React.FC<Props> = ({ eventData }) => {
             {email && <div>{email}</div>}
             {telephone && <div>{telephone}</div>}
             {infoUrl && (
-              <a className={styles.link} href={infoUrl || ""} target="__blank">
+              <a
+                className={styles.link}
+                href={infoUrl || ""}
```

Comment:
infoUrl is guaranteed to be something, it is checked already before rendering this tag